packages/react/src/_popover/Popover.tsx

```diff
@@ -89,10 +96,12 @@ export function usePopover<RT extends ReferenceType = ReferenceType>(
 			// https://floating-ui.com/docs/size
 			size({
 				padding: DEFAULT_OFFSET, // Prevents the floating element hit the edge of the screen
-				apply({ elements, rects }) {
+				apply({ availableHeight, elements, rects }) {
 					Object.assign(elements.floating.style, {
-						height: 'max-content',
-						...(maxHeight !== undefined && { maxHeight: `${maxHeight}px` }),
+						...(fixHeightAsContentHeight && { height: 'max-content' }),
+						...(!fixHeightAsContentHeight && {
+							maxHeight: `${maxHeight || availableHeight}px`,
```

Comment:
```suggestion
						...(fixHeightAsContentHeight ? { height: 'max-content' } : {
							maxHeight: `${maxHeight || availableHeight}px`,
```

Reply:
I don't know if that code will pass prettier, but I guess we'll see 🍿

Reply:
lol it appears not to have. I'll pretty it up 💅